core/runtime/src/main/java/io/quarkus/runtime/logging/LoggingSetupRecorder.java

```diff
@@ -197,11 +197,9 @@ public void close() throws SecurityException {
             categories.forEach(new BiConsumer<String, CategoryConfig>() {
                 @Override
                 public void accept(String categoryName, CategoryConfig config) {
-                    final CategoryBuildTimeConfig buildCategory = isSubsetOf(categoryName, buildConfig.categories);
-                    final Level logLevel = getLogLevel(categoryName, config, categories, buildConfig.minLevel);
-                    final Level minLogLevel = buildCategory == null
-                            ? buildConfig.minLevel
-                            : buildCategory.minLevel.getLevel();
+                    final Level logLevel = getLogLevel(categoryName, categories, c -> c.level, buildConfig.minLevel);
```

Comment:
Not terribly important, but now that this is runtime code, can we get rid of the lambdas?

Reply:
Sure, see also: https://github.com/quarkusio/quarkus/pull/24432#discussion_r830690347

Is there any downside to adding getters to that config classes?

Reply:
Not that I know of

Reply:
Done